src/examples/20-seamless.html

```diff
@@ -42,34 +42,13 @@
 
     <script type="text/javascript">
         jQuery(function() {
-            var stringToRepeat = '<p>Hello, World!<\/p>',
-                html = '';
-
-            for (var i = 0; i < 2; i++) {
-                html += stringToRepeat;
-            }
-            for (var i = 0; i < 5; i++) {
-                // A working image, relative to the iframe, with no height
-                html += stringToRepeat;
-                html += [''
-                    , '<img '
-                    , 'src="../../skins/wymeditor_icon.png'
-                    , '?cache_break=' + i + '"'
-                    , ' />'
-                ].join('');
-            }
-            html += '<p>After the images</p>';
-
             jQuery('.wymeditor').wymeditor({
-                html: html,
+              html: '<p>Some initial content... Type in a few paragraphs and see the editor grow in height.</p>',
```

Comment:
This was very confusing, because the initial document content was  identical to the "more-content" that was filled below the editor. Now it does not confuse.